submittal/src/main/java/com/kyriba/submittal/domain/dto/JobInfoDto.java

```diff
@@ -0,0 +1,60 @@
+package com.kyriba.submittal.domain.dto;
+
+/**
+ * @author M-ABL
+ */
+public class JobInfoDto

```

Comment:
Getters and setters could be replaced with Lombok Data annotation

Reply:
the same for 'domain' classes?

Reply:
This topic will be discussed tomorrow. :)